erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -497,7 +497,7 @@ def _get_operations(bom_no, qty=1):
 
 			for d in bom_traversal:
 				if d.is_bom:
-					operations.extend(_get_operations(d.name, qty=d.exploded_qty))
+					operations.extend(_get_operations(d.name, qty=d.exploded_qty/bom_qty))
```

Comment:
this would've broken nested operations as `exploded_qty` already considers BOM_qty.

Changed the logic to:
- If multi-level BOM => Fetch and add all child operation
- Add all top-level operations regardless of multi-level BOM.